ygen/protogen.go

```diff
@@ -440,6 +470,8 @@ func writeProtoEnums(enums map[string]*yangEnum) ([]string, []error) {
 			continue
 		}
 
+		// Make the name of the enum upper case as is it conventional that a protobuf enum
+		// is completely upper case.
```

Comment:
Swap is with it.
Suggestion: "Make the name of the enum upper case to follow protobuf enum convention."

Reply:
Done.